hardhat.config.js

```diff
@@ -4,6 +4,7 @@ const { gray, yellow } = require('chalk');
 require('@nomiclabs/hardhat-ethers');
 require('@openzeppelin/hardhat-upgrades');
 require('@nomiclabs/hardhat-truffle5');
+require('@nomiclabs/hardhat-waffle');
```

Comment:
Why do we need it?

Reply:
Because standard expect doesn't have this validator:

await expect(feesEscrow.transferToPool()).to.be.`revertedWith`('...')

Reply:
We can use `expectRevert`

Reply:
Repalced